scripts/hooks/prepare-commit-msg

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+STATUS_GO_VERSION_FILE_IS_BEING_COMMITTED=$(git diff --cached --name-only -r | grep status-go-version.json)
+
+if [ -n "$STATUS_GO_VERSION_FILE_IS_BEING_COMMITTED" ]
```

Comment:
Why not invert the if and quit early? Less nesting is always good.
```suggestion
if [[ -z "${STATUS_GO_VERSION_FILE_IS_BEING_COMMITTED}" ]]; then
  return # No status-go-version.json changes found.
fi
```
Also, use double square brackets for Bash.